### PR TITLE
Added double-click (and other small improvements) to Import from Library dialog

### PR DESCRIPTION
**Pull request by derekagorhom**
Fixes Issue #7205 

Adding the double click to the import dialog and also fixing items 1, 2 and 3 in #7205 

@N-thony can you kindly test this PR. 

Also @Patowhiz had some suggestions for the double click in the import From library Dialog

### CODE REVIEW

**Comment by derekagorhom** [comment]
This PR replaces PR #7244

**Review by N-thony — APPROVED** [review]

**Review by rdstern — APPROVED** [review]
@derekagorhom this seems to work fine.

**Comment by Patowhiz** [comment]
> @rdstern could you also test? I think the next thing to do will be to load multiple datasets once.

@derekagorhom  yes, and please note that would require the dialog's redesign and implementation. For instance importing multiple datasets may mean no need to give new data frame name, just like the import dialog.

**Comment by Patowhiz** [comment]
@lloyddewit when looking into this with @derekagorhom, I think it's justified to have a `performClickOk` subroutine in `ucrButtons`? We had to access it's inner controls which didn't seem right to me.

We could use the `ClickOk` event because technically it's just an `AfterClickOk` event.

**Comment by lloyddewit** [comment]
@derekagorhom thanks for this PR. 
Does it fully fix issue #7205 or just partially?

**Review by lloyddewit — APPROVED** [review]

**Comment by derekagorhom** [comment]
> @derekagorhom thanks for this PR. Does it fully fix issue #7205 or just partially?

this PR fully fixes issue #7205.

**Comment by lloyddewit** [comment]
> @lloyddewit when looking into this with @derekagorhom, I think it's justified to have a `performClickOk` subroutine in `ucrButtons`? We had to access it's inner controls which didn't seem right to me.
> 
> We could use the `ClickOk` event because technically it's just an `AfterClickOk` event.

@Patowhiz @derekagorhom Understood. Currently, when the user double clicks the list collection, we pretend that the user has clicked the OK button instead. It would be cleaner if the event handlers for the OK click and list collection double click, both call the same shared function.
I looked in `ucrButtons.vb`. The OK click is handled by `cmdOk_Click()`. This function also raises click events, so even if we extract this code into a new shared function, then we are still tightly coupled to click events.
In summary, there may be arguments for refactoring this, but I don't think this needs to be done in this PR.
So I'll go ahead and merge this PR. I'm happy to discuss if you think further changes are needed.
Thanks

**Comment by lloyddewit** [comment]
> @derekagorhom thanks for this PR. Does it fully fix issue #7205 or just partially?

@derekagorhom If the answer is 'fully', then please close issue #7205, thanks

**Comment by N-thony** [comment]
> > @derekagorhom thanks for this PR. Does it fully fix issue #7205 or just partially?
> 
> @derekagorhom If the answer is 'fully', then please close issue #7205, thanks

@lloyddewit  It doesn't fully fixes #7205, we still need to add the possibility of loading `multiple datasets`. If you close this then we need to open a new issue for the remaining work.
